service/dynamodb/examples_test.go

```diff
@@ -8,1495 +8,536 @@ import (
+// requests.  Only the AlbumTitle attribute is returned.
+func ExampleDynamoDB_BatchGetItem_shared00() {
+	svc := dynamodb.New(session.New())
+	input := &dynamodb.BatchGetItemInput{
+		RequestItems: map[string]*dynamodb.KeysAndAttributes{
+			"Music": {},
 		},
-		ExpressionAttributeNames: map[string]*string{
-			"Key": aws.String("AttributeName"), // Required
-			// More values...
-		},
-		ExpressionAttributeValues: map[string]*dynamodb.AttributeValue{
-			"Key": { // Required
-				B:    []byte("PAYLOAD"),
-				BOOL: aws.Bool(true),
-				BS: [][]byte{
-					[]byte("PAYLOAD"), // Required
-					// More values...
-				},
-				L: []*dynamodb.AttributeValue{
-					{ // Required
-					// Recursive values...
-					},
-					// More values...
-				},
-				M: map[string]*dynamodb.AttributeValue{
-					"Key": { // Required
-					// Recursive values...
-					},
-					// More values...
-				},
-				N: aws.String("NumberAttributeValue"),
-				NS: []*string{
-					aws.String("NumberAttributeValue"), // Required
-					// More values...
-				},
-				NULL: aws.Bool(true),
-				S:    aws.String("StringAttributeValue"),
-				SS: []*string{
-					aws.String("StringAttributeValue"), // Required
-					// More values...
-				},
-			},
-			// More values...
-		},
-		ReturnConsumedCapacity:      aws.String("ReturnConsumedCapacity"),
-		ReturnItemCollectionMetrics: aws.String("ReturnItemCollectionMetrics"),
-		ReturnValues:                aws.String("ReturnValue"),
 	}
-	resp, err := svc.DeleteItem(params)
 
+	result, err := svc.BatchGetItem(input)
 	if err != nil {
-		// Print the error, cast err to awserr.Error to get the Code and
-		// Message from an error.
-		fmt.Println(err.Error())
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			case dynamodb.ErrCodeProvisionedThroughputExceededException:
+				fmt.Println(dynamodb.ErrCodeProvisionedThroughputExceededException, aerr.Error())
+			case dynamodb.ErrCodeResourceNotFoundException:
+				fmt.Println(dynamodb.ErrCodeResourceNotFoundException, aerr.Error())
+			case dynamodb.ErrCodeInternalServerError:
+				fmt.Println(dynamodb.ErrCodeInternalServerError, aerr.Error())
+			default:
+				fmt.Println(aerr.Error())
+			}
+		} else {
+			// Print the error, cast err to awserr.Error to get the Code and
+			// Message from an error.
+			fmt.Println(err.Error())
+		}
 		return
 	}
 
-	// Pretty-print the response data.
-	fmt.Println(resp)
+	fmt.Println(result)
 }
 
-func ExampleDynamoDB_DeleteTable() {
-	sess := session.Must(session.NewSession())
-
-	svc := dynamodb.New(sess)
-
-	params := &dynamodb.DeleteTableInput{
-		TableName: aws.String("TableName"), // Required
+// To add multiple items to a table
+//
+// This example adds three new items to the Music table using a batch of three PutItem
+// requests.
+func ExampleDynamoDB_BatchWriteItem_shared00() {
+	svc := dynamodb.New(session.New())
+	input := &dynamodb.BatchWriteItemInput{
+		RequestItems: map[string][]*dynamodb.WriteRequest{},
```

Comment:
I think these are missing nested values from the example model. Based on the description it sounds like there should be additional fields.